Add a note about ACS and organizations (#2169)
theforeman/foreman-documentation: guides/common/modules/con_managing-alternate-content-sources.adoc

You must refresh the alternate content source after creation or after making any changes.
A weekly cron job refreshes all alternate content sources.
You can also refresh the alternate content sources manually using the {ProjectWebUI} or the Hammer CLI.

> Alternate content sources associated with the internal {SmartProxy} or with {SmartProxies} attached to multiple organizations affect all organizations.

Reviewer: ```suggestion
Alternate content sources associated with your {ProjectServer} or {SmartProxyServers} to multiple organizations affect all organizations.
```

IMHO internal Smart Proxy == Foreman Server; attached Smart Proxy == Smart Proxy.

Author: @maximiliankolb makes sense.

You must refresh the alternate content source after creation or after making any changes.
A weekly cron job refreshes all alternate content sources.
You can also refresh the alternate content sources manually using the {ProjectWebUI} or the Hammer CLI.
Alternate content sources associated with the internal {SmartProxy} will affect all organizations.

> Alternate content sources associated with {SmartProxies} attached to multiple organizations will affect all organizations.

Author: @adamlazik1 I agree. @ianballou is it okay if we keep it as a single sentence?

> Alternate content sources associated with {SmartProxies} attached to multiple organizations will affect all organizations.

Reviewer 2: Single sentence is fine!
